zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -70,9 +71,13 @@
 
   // serialize Paragraph#runtimeInfos and Note#path to frontend but not to note file
   private static final ExclusionStrategy strategy = new ExclusionStrategy() {
+    List<String> fieldsExcluded = Arrays.asList(
+            System.getProperty("zeppelin.note.file.exclude.fields", "")
```

Comment:
We'd better to put it into zeppelin-site.xml ?

Reply:
Added the propery to zeppelin-site.xml template

Reply:
Then we should `ZeppelinConfiguration.getX` of `System.getProperty`

Reply:
Comment addressed